packages/api/src/api/project/post.ts

```diff
@@ -25,6 +26,10 @@ export const post = async (req: Request, res: Response) => {
 
       project = new Project(data);
       project.contributors.add(lockedUser);
+      const hostRes = await axios.get(data.repoLink);
+      if (hostRes.status !== 200) {
+        throw new Error('oh no repoLink cannot be found');
```

Comment:
This will simply break the request and cause a timeout on behalf of the client; this needs to be a structured response to the front end with a meaningful error

```suggestion
        res.status(400).send('Repo URL could not be validated');
        return;
```

Reply:
So even inside of the try block we should just immediately send the response instead of throwing to the catch block?

Reply:
Oh I totally missed that... ok the way yall are doing it is fine but the error message is way too playful for a server; can you sub mine for what you have?